packages/x-components/src/x-modules/experience-controls/store/actions/fetch-controls.action.ts

```diff
@@ -0,0 +1,21 @@
+import { ExperienceControlsXStoreModule } from '../types';
+
+/**
+ * Default implementation for the {@link ExperienceControlsActions.fetchControls}.
+ *
+ * @param _context - The {@link https://vuex.vuejs.org/guide/actions.html | context} of the actions,
+ * provided by Vuex.
+ * @param request - To be done.
+ * @returns To be done.
+ *
+ * @public
+ */
+
+export const fetchControls: ExperienceControlsXStoreModule['actions']['fetchControls'] = () => {
+  return fetch(
+    // eslint-disable-next-line max-len
+    'https://config-service.internal.test.empathy.co/public/configs?service=xcontrols&instance=empathy'
+  ).then(response => response.json());
```

Comment:
To simulate how the adapter (approximately) will work, this should return an object with the controls and the events (currently will be empty, or for testing purposes you can add a random event). Something like:

````
{
  controls: response.json(),
  events: {
    whateverEvent: whateverPayload
  }
}
````